longitudinal_control/Angles angles
float64 vel_GPS
sensor_msgs/NavSatFix GPS_Data
geometry_msgs/Vector3 accel
geometry_msgs/Point position_GPS


